tests/test_sortedset_transport.py

```diff
@@ -8,30 +8,31 @@
 import six
 
 from .utils.fakeredis_ext import FakeStrictRedisWithConnection
+from fakeredis import FakeServer
 from kombu import Connection
 from kombu.five import Empty
-from kombu_redis_priority.transport.redis_priority_async import redis, Transport
+from kombu_redis_priority.transport.redis_priority_async import Transport
 
 
 class TestSortedSetTransport(unittest.TestCase):
-    def setUp(self):
-        self.faker = FakeStrictRedisWithConnection()
-        with mock.patch.object(redis, 'StrictRedis', FakeStrictRedisWithConnection):
-            self.connection = self.create_connection()
-            self.channel = self.connection.default_channel
-
-    def tearDown(self):
-        self.faker.flushall()
```

Comment:
Was this removed because we don't care about state leaking here either?

Reply:
This was removed because each test will get its own brand new fakeredis client and server (see the new `setUp` method). So there really isn't any point in explicitly flushing things after each test.